### DISCUSSION TITLE
Error while placing the order

### DISCUSSION THREAD

**Discussion by naveen3607** [learndevopsonline/learndevopsonline#1232] [Q&A]
Hi Raghu,

I have been trying to setup the roboshop project from start to end. Everything is coming good until placing order. It is showing as "Error Placing Order".

I reviewed every step from start to end and tried three time until now.

Can you please tell me how can i solve this?

Thanks and Regards,
Naveen
![image](https://github.com/learndevopsonline/learndevopsonline/assets/138672193/ec46adeb-5cfc-4a47-a3a9-a32cbea6a143)
![image](https://github.com/learndevopsonline/learndevopsonline/assets/138672193/7e7331b0-159f-4f45-bf95-a968c911bbb3)
![image](https://github.com/learndevopsonline/learndevopsonline/assets/138672193/eade1308-dd1e-48c1-9b1d-5f37836b6fd6)

**Comment by maheshbusireddy** [comment]
@naveen3607 your payment is working fine please check.

**Comment by naveen3607** [comment]
It's working now. Thanks
